saleor/plugins/apps.py

```diff
@@ -0,0 +1,15 @@
+from django.apps import AppConfig
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+
+class PluginConfig(AppConfig):
+    name = "saleor.plugins"
+    verbose_name = "Plugins"
+
+    def ready(self):
+        manager_path = settings.PLUGINS_MANAGER
+        import_string(manager_path)
```

Comment:
There is a Django checker (see `check_plugins` function) that validates plugin paths in settings. Not sure when AppConfig is run, but I'm wondering if this solution is compatible with the checker? The checker should run first.

Reply:
I will move this validation from checks to AppConfig.ready.

Reply:
IIRC AppConfig.ready is called before the checks. Not sure we need that checker, though, as long as it's clear which module failed to load.